common/djangoapps/student/views.py

```diff
@@ -1754,6 +1755,9 @@ def create_account_with_params(request, params):
             }
         )
 
+    # Announce registration
+    REGISTER_USER.send(sender=None, user=user, profile=profile)
```

Comment:
Re: circular imports... I'd love to have the `email_marketing` importing from `student`, and not the other way around.